yarn.lock

```diff
@@ -0,0 +1,7716 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We've been using NPM to this point. I don't think we want to have two package managers in the same project.

Reply:
I just assumed we were using `yarn` due to the [README.md](https://github.com/wijohnst/thriving-park-documentation#readme) instructions

Reply:
Removed in 4906684

Reply:
I need to update the README for this repo. Its set to the default from Docusaurus. Sorry about that!

Reply:
Created an issue to update readme:

#27